### ISSUE TITLE
Wiki: The JSON API page doesn't explain what a link is

### ISSUE THREAD

**Issue by guoarthur** [reddit/reddit#1788]
Page in question: https://github.com/reddit/reddit/wiki/JSON

Recently got started in looking at the Reddit API. So sorry in advance if this is stupid :)  

I was trying to figure out how I might find the OP's username given I have a comment id in a thread. I was trying to figure out 1. If a link is what I'm looking for to find OP. 2. If link_id or parent_id was what I needed. I didn't know that a link was essentially a submission, and it took me a while to figure that out...

Possible changes:

* Change the line ` link (implements votable | created) ` to ` link (implements votable | created) \n AKA submission, thread or post ` (This might screw up the format, but would be more beginner friendly)

* Move the comment data structure below the link data structure (since a link "has many" comments)

**Comment by michaelborchardt** [comment]
Since 'thing' is the base class in the Reddit API, you can just make a call to the thing object and pass the id in, and it will most likely respond with the proper 'thing', which in this case would be the comment with the corresponding id. I've never worked with the API, but I'm assuming that since every object inherits from the thing, this is the case. Once you have the comment, just call the authors name from the comment object.

**Comment by guoarthur** [comment]
Yep, I figured out how to do it in a different way. Just wanted to post about my experiences and maybe  change the wiki
